### PR TITLE
Adding media association to sproperties

### PR DESCRIPTION
**Pull request by t2oh4e**
This will add a new entry media in the sPropeties Array. Useful if you want to display the images associated to the properties in the frontend.

### CODE REVIEW

File: engine/Shopware/Components/Compatibility/LegacyStructConverter.php [hunk 1]
```diff
@@ -597,6 +597,14 @@ function (StoreFrontBundle\Struct\Property\Option $option) {
                 },
                 $group->getOptions()
             );
+            
+            $mediaValues = array();
+            foreach($group->getOptions() as $option) {
```
Comment (bcremer) [inline engine/Shopware/Components/Compatibility/LegacyStructConverter.php]: Please add a space between `foreach` and the opening parenthesis.

**Comment by t2oh4e** [comment]
Space added.

**Comment by bcremer** [comment]
@t2oh4e The fix looks good. Can you please redo the commits using a real name a git author.

**Comment by t2oh4e** [comment]
Whoops... Looks like I messed up my Shopware fork... I will do this PR again later with a real name.

**Pull request closed without merging**